### DISCUSSION TITLE
Why I can't install the scripts on my server?

### DISCUSSION THREAD

**Discussion by levaweb** [katspaugh/wavesurfer.js#3204] [Q&A] [answered]
A silly question please,
I've added the scripts in (A) and it works.

Then I've installed ws locally (below), and moved all scrips*.js to my server (B)
But I get the error "Cannot use import statement outside a module"

>npm install --save wavesurfer.js 

A | OK
```
<script src="https://unpkg.com/wavesurfer.js@7"></script>
<script src="https://unpkg.com/wavesurfer.js@7/dist/plugins/regions.min.js"></script>
<script src="https://unpkg.com/wavesurfer.js@7/dist/plugins/timeline.min.js"></script>
```

B | Error
```
<script src="assets/js/wavesurfer/wavesurfer.js"></script>
<script src="assets/js/wavesurfer/plugins/regions.min.js"></script>
<script src="assets/js/wavesurfer/plugins/timeline.min.js"></script>
```

Did I miss anything?

Thanks

**Comment by katspaugh** [comment, marked as answer]
assets/js/wavesurfer/wavesurfer.js is wrong. You should be inserting a .min.js file.

All kinds of questions are welcome, although I’d appreciate if they are about wavesurfer and not generic JS questions that are easy to debug/google. Cheers.

**Reply by levaweb** [reply]
Thanks ,it works
